src/adapters/AloeVolatilityOracleAdapter.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: BUSL 1.1
+pragma solidity 0.8.13;
+
+import "../interfaces/IAloeVolatilityOracleAdapter.sol";
+
+import "../libraries/aloe/interfaces/IVolatilityOracle.sol";
+import "../libraries/aloe/VolatilityOracle.sol";
+
+import "v3-core/contracts/interfaces/IUniswapV3Factory.sol";
+import "v3-core/contracts/interfaces/IUniswapV3Pool.sol";
+
+import "../utils/Keep3rV2Job.sol";
+
+/**
+ * @notice This contract adapts the Aloe capital volatility oracle
+ * contract from https://github.com/aloelabs/aloe-blend.
+ */
+contract AloeVolatilityOracleAdapter is IAloeVolatilityOracleAdapter, Keep3rV2Job {
```

Comment:
We should rename this to `ValoremVolatilityOracleAdapter`, or just `VolatilityOracleAdapter`